code/stable/glassbr/SRS/GlassBR_SRS.tex

```diff
@@ -75,15 +75,15 @@ \subsection{Table of Symbols}
 \\
 $b$ & Plate width (short dimension) & m
 \\
-$capacity$ & Capacity or load resistance & Pa
+$capacity$ & Capacity & Pa
```

Comment:
I prefer the old definition of capacity.  The new one does not say anything that the variable name itself does not already convey.